src/platform_impl/web/web_sys/canvas/pointer_handler.rs

```diff
@@ -22,6 +26,10 @@ impl PointerHandler {
             on_cursor_move: None,
             on_pointer_press: None,
             on_pointer_release: None,
+            on_pointer_move: None,
+            on_pointer_down: None,
+            on_pointer_up: None,
+            on_pointer_cancel: None,
         }
```

Comment:
Rename to `on_touch_(move|down|up|cancel)`

Reply:
Renamed